docfx-util.ps1

```diff
@@ -0,0 +1,57 @@
+
+[CmdletBinding(PositionalBinding=$false)]
+param 
+(
+    [parameter(mandatory=$false)][switch][Alias('c')]$clean,
+    [parameter(mandatory=$false)][switch][Alias('d')]$doclinkchecker,
+    [parameter(mandatory=$false)][string][Alias('l')]$lychee,
+    [parameter(mandatory=$false)][string][Alias('a')]$all
+)
+
+
+
+# this is called removeartifacts instead of clean because clean might be taken up by another name
+function removeartifacts
+{
+    $deletePaths = '.\workflows\*.svg', '.\api\*.yml', '.\api\.manifest', '.\_site\', '.\_raw\', '.\_view\'
+    foreach($deletePath in $deletePaths)
+    {    
+        if (Test-Path $deletePath) {
+            Remove-Item $deletePath -Recurse
+        } 
+    }
+}
+
+function doclinkchecker
+{
+    .\build.ps1 --logLevel Warning --warningsAsErrors
```

Comment:
Due to how long these calls can take to run, can we add some info that is printed before and between the commands describing what is happening? e.g., before this say something like "docfx-util.ps1: Beginning build process...\n\n" to denote where it is coming from and what is happening. The extra carriage returns are to differentiate the output from the output of the build.ps1 command. Maybe even having a line break (----------------------------------------) to really show which step is being run.